lib/repositories/implementation/genres_repository.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+
+import '../interfaces/genres_repository_interface.dart';
+
+class GenresRepository extends GenresRepositoryInterface {
+  static const mockDataGenres = 'mock_data/genres.json';
+  late Map<int, String> genresById = {};
+
+  Future<Map<int, String>> parseJsonGenres() async {
+    final jsonPath = await rootBundle.loadString(mockDataGenres);
+    Map<String, dynamic> json = jsonDecode(jsonPath);
+    return {
+      for (var genre in json['genres']) genre['id'] as int: genre['name']
```

Comment:
avoid the use of type cast when you don't need it

Reply:
Okay, fixed

Reply:
Done!